Update security.md (#4594)
centreon/centreon-documentation: versioned_docs/version-24.10/security/security.md

penetration tests are run on a yearly basis to independently identify vulnerabilities and tune best
practice recommendations for safe implementation.

## Vulnerability publication

> To mark its commitment to security and vulnerability management, Centreon is a [CNA](https://www.cve.org/Media/News/item/news/2025/02/11/Centreon-Added-as-CNA) (CVE Numbering Authority). Centreon is now authorized to assign CVE identifiers to vulnerabilities affecting its products, including open-source software components. This designation ensures a more efficient and transparent process for identifying, managing, and disclosing security vulnerabilities. 

Reviewer: has been designated as CNA?

Author: "has become" à la limite mais on n'a pas été "désigné" : il y a un process d'approbation

> To mark its commitment to security and vulnerability management, Centreon is a [CNA](https://www.cve.org/Media/News/item/news/2025/02/11/Centreon-Added-as-CNA) (CVE Numbering Authority). Centreon is now authorized to assign CVE identifiers to vulnerabilities affecting its products, including open-source software components. This designation ensures a more efficient and transparent process for identifying, managing, and disclosing security vulnerabilities. 

Reviewer: ```suggestion
To mark its commitment to security and vulnerability management, Centreon has become a [CNA](https://www.cve.org/Media/News/item/news/2025/02/11/Centreon-Added-as-CNA) (CVE Numbering Authority). Centreon is now authorized to assign CVE identifiers to vulnerabilities affecting its products, including open-source software components. This designation ensures a more efficient and transparent process for identifying, managing, and disclosing security vulnerabilities. 
```

This role enables Centreon to swiftly address potential security issues, keeping users and IT professionals informed while enhancing the overall security posture of its solutions.

> Centreon also publish security bulletin to communicate about security fixes : 

Reviewer: ```suggestion
Centreon also publishes security bulletins to communicate about security fixes: 
```

Subscribe to the [Security Bulletins thread](https://thewatch.centreon.com/latest-security-bulletins-64) on our community platform The Watch to keep up with the latest vulnerability fixes.

### Vulnerability identification

> Managing vulnerabilities means detecting and classifying them. To detect some tools exists

Reviewer: ```suggestion
Managing vulnerabilities means detecting and classifying them. To detect them, certain tools exist
in the Centreon environment:
```

> in Centreon environment :

Reviewer: ```suggestion
```

> ● SAST (Static application security testing) : This is a built-in CI/CD functionality that

Reviewer: ```suggestion
* SAST (Static application security testing) : This is a built-in CI/CD functionality that
```

> scans 100% of codebase but also dependencies. It discovers vulnerabilities earlier in

Reviewer: ```suggestion
scans 100% of the codebase but also dependencies. It discovers vulnerabilities early in
```

the SDLC process.

> ● Infrastructure Vulnerability scanning : Quite same as DAST but related to

Reviewer: ```suggestion
* Infrastructure Vulnerability scanning : Quite the same as DAST but related to
```

> infrastructure : Servers, VM, containers...

Reviewer: ```suggestion
infrastructure: Servers, VM, containers...
```

● Bug Bounty Program & Public vulnerability disclosure program
● Penetration testing

Upon receipt of a report for a vulnerability, Centreon will attempt to replicate the reported
vulnerability and build an action plan with timelines depending on the computed severity.

Centreon will provide feedback to the reporter of the vulnerability and work with them to mitigate or fix
the vulnerability.

### Vulnerability scoring

The discovered vulnerabilities are analyzed and assessed to rule out false positives, identify

> the issues that pose the most risk to Centreon and remediation that have less impact on Centreon’s

Reviewer: ```suggestion
the issues that pose the most risk to Centreon and remediation that has the least impact on Centreon’s
```

environment.
The classification must follow the CVSS v3:

| CVSS score           | Remediation plan                                                                                                                                                                                      |
|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | 9.0 - 10 (Critical)  | This is  a blocker and  must be taken into account immediately by the team; this will require a hotfix. |

Reviewer: ```suggestion
| 9.0 - 10 (Critical)  | This is a blocker and  must be taken into account immediately by the team; this will require a hotfix. |
```

| 7.0 - 8.9 (High)     | Fix or workaround within 30 days                                                                         |

> | 4.0 - 6.9 (Medium)   | Next planned release : 6 month (typically  within 6 months)                                                                                                  |

Reviewer: ```suggestion
| 4.0 - 6.9 (Medium)   | Next planned release : 6 months (typically within 6 months)                                                                                                  |
```

> | 0.1 – 3.9 (Low)      | 12 monthes                                                                    |

Reviewer: ```suggestion
| 0.1 – 3.9 (Low)      | 12 months                                                                    |
```

> ### Vulnerability disclosure program

Reviewer: ```suggestion
### Vulnerability disclosure program

```

> Centreon follows the reponsible disclosure policy : 

Reviewer: ```suggestion
Centreon follows a reponsible disclosure policy : 
```

> ● Denial of service (DoS) attacks on Centreon applications, servers, networks or infrastructure

Reviewer: ```suggestion
* Denial of service (DoS) attacks on Centreon applications, servers, networks or infrastructure
```

are strictly forbidden.

> ● Avoid tests that could cause degradation or interruption of our services.

Reviewer: ```suggestion
* Avoid tests that could cause degradation or interruption of our services.
```

> ● Do not use automated scanners or tools that generate large amount of network traffic.

Reviewer: ```suggestion
* Do not use automated scanners or tools that generate large amounts of network traffic.
```

> ● Do not leak, manipulate, or destroy any user data or files in any of our applications/servers.

Reviewer: ```suggestion
* Do not leak, manipulate, or destroy any user data or files in any of our applications/servers.
```

> ● Do not copy any files from our applications/servers and disclose them.

Reviewer: ```suggestion
* Do not copy any files from our applications/servers and disclose them.
```

> ● No vulnerability disclosure, full, partial or otherwise, is allowed.

Reviewer: ```suggestion
 @* No vulnerability disclosure, full, partial or otherwise, is allowed.
```

> ● No vulnerability disclosure, full, partial or otherwise, is allowed.

Reviewer: ```suggestion
```

> As an opensource project, Centreon allows vulnerability disclosure from the community. The

Reviewer: ```suggestion
As an open source project, Centreon allows vulnerability disclosure from the community. The
```

penetration tests are run on a yearly basis to independently identify vulnerabilities and tune best
practice recommendations for safe implementation.

## Vulnerability publication

To mark its commitment to security and vulnerability management, Centreon has become a [CNA](https://www.cve.org/Media/News/item/news/2025/02/11/Centreon-Added-as-CNA) (CVE Numbering Authority). Centreon is now authorized to assign CVE identifiers to vulnerabilities affecting its products, including open-source software components. This designation ensures a more efficient and transparent process for identifying, managing, and disclosing security vulnerabilities. 
This role enables Centreon to swiftly address potential security issues, keeping users and IT professionals informed while enhancing the overall security posture of its solutions.

> Centreon also publishes security bulletins to communicate about security fixes: 

Reviewer 2: ```suggestion
Centreon also publishes security bulletins to communicate about security fixes. 
```

Subscribe to the [Security Bulletins thread](https://thewatch.centreon.com/latest-security-bulletins-64) on our community platform The Watch to keep up with the latest vulnerability fixes.

### Vulnerability identification

Managing vulnerabilities means detecting and classifying them. To detect them, certain tools exist
in the Centreon environment:
* SAST (Static application security testing) : This is a built-in CI/CD functionality that
scans 100% of the codebase but also dependencies. It discovers vulnerabilities early in
the SDLC process.

> * Infrastructure Vulnerability scanning : Quite the same as DAST but related to

Reviewer: ```suggestion
* Infrastructure vulnerability scanning : quite the same as DAST but related to
```

> infrastructure: Servers, VM, containers...

Reviewer: ```suggestion
infrastructure: servers, VM, containers...
```

> ● Bug Bounty Program & Public vulnerability disclosure program

Reviewer: ```suggestion
* Bug Bounty Program & Public vulnerability disclosure program
```

> ● Penetration testing

Reviewer: ```suggestion
* Penetration testing
```

Upon receipt of a report for a vulnerability, Centreon will attempt to replicate the reported
vulnerability and build an action plan with timelines depending on the computed severity.

> Centreon will provide feedback to the reporter of the vulnerability and work with them to mitigate or fix

Reviewer 2: ```suggestion
Centreon will provide feedback to the reporter of the vulnerability and work with them to mitigate or to fix
```

environment.
The classification must follow the CVSS v3:

| CVSS score           | Remediation plan                                                                                                                                                                                      |
|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 9.0 - 10 (Critical)  | This is a blocker and  must be taken into account immediately by the team; this will require a hotfix. |
| 7.0 - 8.9 (High)     | Fix or workaround within 30 days                                                                         |
| 4.0 - 6.9 (Medium)   | Next planned release : 6 months (typically within 6 months)                                                                                                  |
| 0.1 – 3.9 (Low)      | 12 months                                                                    |

### Vulnerability disclosure program

> Centreon follows a reponsible disclosure policy : 

Reviewer 2: ```suggestion
Centreon follows a reponsible disclosure policy: 
```

* Do not use automated scanners or tools that generate large amounts of network traffic.
* Do not leak, manipulate, or destroy any user data or files in any of our applications/servers.
* Do not copy any files from our applications/servers and disclose them.

As an open source project, Centreon allows vulnerability disclosure from the community. The
vulnerability disclosure program is implemented and must be followed for the disclosure to be taken
into account. Centreon is committed to the security of its product and services and is continuously
improving policies, processes, and products to meet the highest standards.
Despite our best efforts, due to the highly complex and sophisticated nature of our products and
services, vulnerabilities and errors may still be present in our products and services.
This policy describes Centreon's approach to requesting and receiving reports related to potential
vulnerabilities and errors in its products and services.
Customers, users, researchers, partners and any other person that interacts with Centreon's
products and services are encouraged to report identified vulnerabilities and errors by using the

> form present on the Vulnerability Disclosure Policy platform.

Reviewer 2: This is quite a long paragraph, and it's a bit strange to read: it feels like the sentences are not in the right order. Actually, this applies to the whole VDP section. Can we completely change the order of the sentences and paragraphs so that it feels more fluid? (all sentences are fine if taken separately, it's just the order that can improve things)

Reviewer: @vuntz can you post a suggestion?

Reviewer 2: For instance:

### Vulnerability disclosure program

Centreon is committed to the security of its product and services and is continuously improving policies, processes, and products to meet the highest standards. Despite our best efforts, due to the highly complex and sophisticated nature of our products and services, vulnerabilities and errors may still be present in our products and services.

As an open source project, Centreon welcomes vulnerability disclosure from the community with a process governed by the vulnerability disclosure policy available at this URL: [Vulnerability disclosure policy](https://vdp.centreon.com/p/centreon-VDP). This policy describes Centreon's approach to requesting and receiving reports related to potential vulnerabilities and errors in its products and services. Customers, users, researchers, partners and any other person that interacts with Centreon's products and services are encouraged to report identified vulnerabilities and errors by using the form present on the Vulnerability Disclosure Policy platform.

Furthermore, Centreon follows a reponsible disclosure policy: 
* Denial of service (DoS) attacks on Centreon applications, servers, networks or infrastructure are strictly forbidden.
* Avoid tests that could cause degradation or interruption of our services.
* Do not use automated scanners or tools that generate large amounts of network traffic.
* Do not leak, manipulate, or destroy any user data or files in any of our applications/servers.
* Do not copy any files from our applications/servers and disclose them.

The vulnerability disclosure program is implemented and must be followed for the disclosure to be taken into account.

> Centreon’s public vulnerability disclosure policy is available following this URL :

Reviewer 2: ```suggestion
Centreon’s public vulnerability disclosure policy is available at this URL:
```
